aten/src/ATen/native/mps/operations/Sorting.mm

```diff
@@ -0,0 +1,80 @@
+#include <ATen/ATen.h>
+#include <ATen/Tensor.h>
+#include <ATen/Utils.h>
+#include <ATen/mps/MPSStream.h>
+#include <ATen/native/mps/OperationUtils.h>
+#include <torch/library.h>
+#include <c10/util/Optional.h>
+#include <ATen/native/mps/MPSGraphVenturaOps.h>
+
+namespace at {
+namespace native {
+
+Tensor argsort_stable(const Tensor & self, bool stable, int64_t dim, bool descending) {
```

Comment:
This should be argsort_stable_mps 
```suggestion
Tensor argsort_stable_mps(const Tensor & self, bool stable, int64_t dim, bool descending) {
```

Reply:
You need to use the same name you have declared in the YAML file.

Reply:
Thanks for your comment! I fixed the function name

https://github.com/pytorch/pytorch/pull/87572/commits/bd8381d6abc3f5e1411d968fcb01250e682e26b4